spec/lib/zxcvbn_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/zxcvbn_tester"
+
+describe Zxcvbn do
+  describe '#src' do
+    it 'defaults to library from npm' do
+      default = File.expand_path("#{Rails.root}/node_modules/zxcvbn/dist/zxcvbn.js", __FILE__)
+
+      data_path = Zxcvbn::Tester.new.data_path
+
+      expect(data_path).to eq default
+    end
+  end
```

Comment:
could we also get a test of the localization feature? maybe add to the existing form_password_validator spec?